weblate/trans/util.py

```diff
@@ -395,3 +395,8 @@ def count_words(string: str, language: Language | None = None) -> int:
                 even = not even
         return count
     return sum(len(s.split()) for s in split_plural(string) if not is_unused_string(s))
+
+
+def list_to_tuples(strings: list[str]) -> list[tuple]:
```

Comment:
Make the type annotations more generic on input and specific on output:

```suggestion
def list_to_tuples(strings: Iterable[str]) -> list[tuple[str]]:
```